bundles/org.openhab.core/src/test/java/org/eclipse/smarthome/core/util/HexUtilsTest.java

```diff
@@ -1,3 +1,4 @@
+package org.eclipse.smarthome.core.util;
```

Comment:
Your IDE probably tried to "help" you with moving this package declaration above the header. Can you move it back below the copyright header?

Reply:
done